scripts/cibuild.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+vagrant up --provision
+vagrant ssh -c "
+    cd /vagrant
+    echo -e '\nRunning linter...'
+
+    # run linter with console output
+    npm run lint
```

Comment:
That makes sense for projects that need it, but this is a single-project repository, which is why all this scripting seems excessive. Testing and building are both one-line commands, typically run outside the VM during development.